### DISCUSSION TITLE
Phrase search returns stemmed results

### DISCUSSION THREAD

**Discussion by Meesch** [UUDigitalHumanitieslab/I-analyzer#1580] [General]
When the user uses a phrase search, i.e. a query with quotation marks around it, like for example "friends and family", I-Analyzer returns results that are stemmed and also have stopwords removed, see below:
![Screenshot 2023-07-21 at 11 42 56](https://github.com/UUDigitalHumanitieslab/I-analyzer/assets/31687030/05d92c2a-b998-4923-bbcd-f838a7e22cd5)
_Query "friends and family"_

![Screenshot 2023-07-21 at 11 43 37](https://github.com/UUDigitalHumanitieslab/I-analyzer/assets/31687030/3faa79ba-2416-41e8-8e47-48784cbc1ba3)
_Result based on 'their friends, their family'_

I think this is not the behaviour that the user expects, since phrase search is a way to search exact phrases, stopwords and all: should we change this in the tool?

**Comment by lukavdplas** [comment]
That may be difficult to execute, as the query `government "friends and family"` should then search for `government` in the stemmed field but for `"friends and family"` in the exact field. Gets even more difficult with something like `government "friends and family" society disease ~2`. Handling that is technically possible, but only if we write our own simple query syntax -> query DSL parser, which I'm not keen on.

In any case, I don't know if this is necessarily a problem - it can also be quite useful to search for phrases in a field with stopwords removed. I think your screenshots actually show a nice example of that. That said, whether stemming or stopword removal happens is not communicated very clearly, which can make this more confusing.

**Reply by lukavdplas** [reply]
> That said, whether stemming or stopword removal happens is not communicated very clearly, which can make this more confusing.

See also #1716 , which I think also shows an example of stopword removal being communicated very poorly. It's not at all intuitive that the way to turn off stopword/stemming analysis for the query is to select the "speech" field. (And no amount of documentation is going to make that seem logical.)

Since the different multifields are all presented as one to the user, it just doesn't make sense that you choose analysis in the field selection. Perhaps we could present this as a separate option, something like this:

![screenshot of I-analyzer, showing collapsible "advanced options" below the search bar, with checkboxes to toggle stopword analysis and stemming](https://github.com/user-attachments/assets/841049c2-8755-4109-8aba-08967ec5c66b)

**Comment by lukavdplas** [comment]
Just a thought: https://github.com/UUDigitalHumanitieslab/I-analyzer/issues/1099 could provide clarity for cases like this.
